amical/tools.py

```diff
@@ -79,7 +79,7 @@ def find_max(img, filtmed=True, f=3):
     return X, Y
 
 
-def crop_max(img, dim, offx=0, offy=0, filtmed=True, f=3):
+def crop_max(img, dim, iframe=0, offx=0, offy=0, filtmed=True, f=3):
```

Comment:
this is an unwarranted breaking change: new arguments should always be added at the end of an existing signature, and preferably be made keyword-only